service/create/service.go

```diff
@@ -42,6 +43,13 @@ type Config struct {
 	Viper *viper.Viper
 }
 
+// certificateSecret stores a cert issued by Vault that will be stored as a k8s secret.
```

Comment:
I made this private but maybe it should be public if we want to expose issue cert as a library?